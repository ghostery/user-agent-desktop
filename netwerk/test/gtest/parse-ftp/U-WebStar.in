ftp> open ftp.webstar.com
Connected to ftp.webstar.com.
220 FTP server ready.
Name (ftp.webstar.com:cyp): 
331 Anonymous login OK, send login as password.
230 User logged in.
Remote system type is UNIX.
ftp> pwd
257 "/" is the current directory.
ftp> syst
215 UNIX WebSTAR FTP.
ftp> cd products
250 CWD OK, "/products".
ftp> cd WebStar
250 CWD OK, "/products/WebStar".
ftp> cd Updates
250 CWD OK, "/products/WebStar/Updates".
ftp> ls
200 PORT OK, IP address 134.93.247.34 port 49247
150 Opening data connection.
drwx------  0  owner  group         1 Jan 30 17:16 bundleup
drwx------  0  owner  group         2 Nov 08  2001 docs
drwx------  0  owner  group        12 Nov 08  2001 extending_webstar
drwx------  0  owner  group         2 Apr 25 09:40 Products
drwx------  0  owner  group         1 Jan 30 16:20 updates
drwx------  0  owner  group        11 Jan 30 17:16 webstar_dev
drwx------  0  owner  group         2 May 01 15:46 WebSTAR
drwx------  0  owner  group        15 May 13 09:54 Installers
drwx------  0  owner  group         8 Apr 25 09:59 Updates
drwx------  0  owner  group         7 Apr 25 10:00 Plug-ins
drwx------  0  owner  group         2 Apr 06  2001 SSL
-rwx------  0  owner  group   1605421 Mar 30  2001 webstar1.3.2_updater.sea.hqx
-rwx------  0  owner  group   6883859 Mar 30  2001 WebSTAR211installer.sea.hqx
-rwx------  0  owner  group   7366223 Mar 30  2001 WebSTAR211updater.sea.hqx
-rwx------  0  owner  group  13415184 Mar 30  2001 WebSTAR302updater.sea.hqx
drwx------  0  owner  group         4 Apr 05  2001 WebSTAR_44
drwx------  0  owner  group         2 Apr 25 10:00 WebSTAR_V
226 Transfer complete.
ftp> close
221 Goodbye.
ftp> 