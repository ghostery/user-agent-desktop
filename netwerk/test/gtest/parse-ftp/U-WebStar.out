<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

01-30-2002  17:16:00       <DIR>          bundleup
11-08-2001  00:00:00       <DIR>          docs
11-08-2001  00:00:00       <DIR>          extending_webstar
04-25-2002  09:40:00       <DIR>          Products
01-30-2002  16:20:00       <DIR>          updates
01-30-2002  17:16:00       <DIR>          webstar_dev
05-01-2002  15:46:00       <DIR>          WebSTAR
05-13-2002  09:54:00       <DIR>          Installers
04-25-2002  09:59:00       <DIR>          Updates
04-25-2002  10:00:00       <DIR>          Plug-ins
04-06-2001  00:00:00       <DIR>          SSL
03-30-2001  00:00:00              1605421 webstar1.3.2_updater.sea.hqx
03-30-2001  00:00:00              6883859 WebSTAR211installer.sea.hqx
03-30-2001  00:00:00              7366223 WebSTAR211updater.sea.hqx
03-30-2001  00:00:00             13415184 WebSTAR302updater.sea.hqx
04-05-2001  00:00:00       <DIR>          WebSTAR_44
04-25-2002  10:00:00       <DIR>          WebSTAR_V
